### PR TITLE
fix: audio plays in fast forward for first few seconds

### PR DESCRIPTION
**Pull request by felix920506**
## Summary

Audio may in some cases play in fast forward for the first few seconds when the source is slow to start, such as ffmpeg

## Information

- [x] This PR fixes an issue.
- [ ] This PR adds something new (e.g. new method or parameters).
- [ ] This PR is a breaking change (e.g. methods or parameters removed/renamed).
- [ ] This PR is **not** a code change (e.g. documentation, README, typehinting,
      examples, ...).

## Checklist

- [x] I have searched the open pull requests for duplicates.
- [x] If code changes were made then they have been tested.
  - [ ] I have updated the documentation to reflect the changes.
- [ ] If `type: ignore` comments were used, a comment is also left explaining why.
- [x] I have updated the changelog to include these changes.

### CODE REVIEW

**Review by Lulalaby — APPROVED** [review]

File: CHANGELOG.md [hunk 1]
```diff
@@ -29,6 +29,8 @@ These changes are available on the `master` branch, but have not yet been releas
   ([#2577](https://github.com/Pycord-Development/pycord/pull/2577))
 - Fixed `codec` option for `FFmpegOpusAudio` class to make it in line with
   documentation. ([#2581](https://github.com/Pycord-Development/pycord/pull/2581))
+- Fixed audio playing back in fast forward for the first few seconds sometimes.
```
Comment (JustaSqu1d) [inline CHANGELOG.md:32]: Please use more clear language.

**Review by JustaSqu1d — CHANGES_REQUESTED** [review]
I have no idea what you're trying to say

File: CHANGELOG.md [hunk 2]
```diff
@@ -29,6 +29,8 @@ These changes are available on the `master` branch, but have not yet been releas
   ([#2577](https://github.com/Pycord-Development/pycord/pull/2577))
 - Fixed `codec` option for `FFmpegOpusAudio` class to make it in line with
   documentation. ([#2581](https://github.com/Pycord-Development/pycord/pull/2581))
+- Fixed audio plays in fast forward at beginning of audio files.
```
Comment (Lulalaby) [inline CHANGELOG.md:32]:
```suggestion
- Fixed a possible bug where audio would play too fast at the beginning of audio files.
```

**Review by Lulalaby — COMMENTED** [review]
@JustaSqu1d might this be better?

File: CHANGELOG.md (hunk 2 above)
Comment (JustaSqu1d) [reply CHANGELOG.md:32]: Yeah, much better

**Review by Lulalaby — APPROVED** [review]

**Review by JustaSqu1d — APPROVED** [review]

**Pull request merged**